doc/howto/network_ipam.md

```diff
@@ -0,0 +1,48 @@
+(network-ipam)=
+# How to display IPAM information of a LXD deployment
+
+{abbr}`IPAM (IP address management)` is a method used to plan, track, and manage the information associated with a computer network's IP address space. In essence, it's a way of organizing, monitoring, and manipulating the IP space in a network.
+
+You can display {abbr}`IPAM` information like so:
+
+```bash
+lxc network list-allocations
+
+Flags:
+    --all-projects          Run against all projects
+    -p, --project string    Run again a specific project (default "default")
+
+```
+
+The resulting output will look something like this:
+
+```
+{
+	"cidr_addresses": [
+		"10.6.105.1/32",
+		"fd42:3cce:990:a1fd::1/128"
```

Comment:
Can you fake the IPs to use [documentation IP addresses](https://en.wikipedia.org/wiki/Reserved_IP_addresses), or does that not make sense here?